nextflow.config

```diff
@@ -12,6 +12,23 @@ params {
     // Input options
     input                      = null
 
+    // Taxonomy classification options
+    run_taxonomic_classification            = false
+    taxonomic_classification_tool           = 'mmseqs2'
+
+    mmseqs_databases_localpath              = null
```

Comment:
I still wonder about this and I would lean to be more generic to account for future tools (but we can vote if there is a disagreement), presumably other taxonomic tools will require a database... for generic parameters I would stick with `--taxonomic_classification_databasepath`,  `--taxonomic_classification_savetmp`, etc.

Then mmseqs ones would be:

```
--taxonomic_classification_mmseqs_savetmp
--taxonomic_classification_mmseqs_searchtype
--taxonomic_classification_mmseqs_taxlineadge
```

etc.

Reply:
but again mmseqs has many submodules. so we should also include the submodule in the flag name. so for your suggestion that would be '--taxonomic_classification_mmseqs_databases_taxlineadge', which i find is sooo long. Also i dont see ourselves adding any other additional taxonomic classification tool soon ;) so their shouldnt be any conflict